grpc_publisher_service.py

```diff
@@ -513,6 +514,83 @@ def get_platform_info(platform_letter):
                     send_to_sentry=True,
                 )
             return platform_info, None
+        
+        def handle_test_client(
+            msisdn,
+            payload,
+            context,
+            response,
```

Comment:
The context, and response don't need to be args, they're already globally availiable